tests/storage/lvm_test.py

```diff
@@ -1059,6 +1059,37 @@ def test_vg_get_error(fake_devices):
     assert "Fake lvm error" in e.value.value
 
 
+def test_vg_get_vgs_with_non_existing_vg(fake_devices):
```

Comment:
Better test this using lvm.getVGs() with real vgs. If we cannot covert all the interesting cases you can add a test using fakes.

We already test this in https://github.com/oVirt/vdsm/blob/013692737dc9dd89b977cb8a91e479c8757fd0db/tests/storage/lvm_test.py#L1255
so we may not need another test.

Reply:
I will remove it then.

Last week I was having issues running sudo tests, so I pushed to increase coverage with non-sudo tests.